---
layout: post
title: New Paper Published!!
subtitle: BMB Rep. 2025 Apr 21;58(5):203â€“208; https://doi.org/10.5483/BMBRep.2024-0020
categories: [ETC]
tags: [Paper, 3D genome, Chromatin interaction, Deep learning, Hi-C sequencing, Machine learning]
comments: true
---

## Unraveling the three-dimensional genome structure using machine learning

<br/>

### Abstract
The study of chromatin interactions has advanced considerably with technologies such as high-throughput chromosome conformation capture (Hi-C) sequencing, providing a genome-wide view of physical interactions within the nucleus.
These techniques have revealed the existence of hierarchical chromatin structures such as compartments, topologically associating domains (TADs), and chromatin loops, which are crucial in genome organization and regulation.
However, identifying and analyzing these structural features require advanced computational methods.
In recent years, machine learning approaches, particularly deep learning, have emerged as powerful tools for detecting and analyzing structural information.
In this review, we present an overview of various machine learning-based techniques for determining chromosomal organization.
Starting with the progress in predicting interactions from DNA sequences, we describe methods for identifying various hierarchical structures from Hi-C data.
Additionally, we present advances in enhancing the chromosome contact frequency map resolution to overcome the limitations of Hi-C data.
Finally, we identify the remaining challenges and propose potential solutions and future directions.

Keywords: 3D genome, Chromatin interaction, Deep learning, Hi-C sequencing, Machine learning

<br/>

  * [Click to view(Paper)](https://doi.org/10.5483/BMBRep.2024-0020)
